h/views/api/groups.py

```diff
@@ -183,7 +174,7 @@ def remove_member(context: GroupContext, request):
     permission=Permission.Group.MEMBER_ADD,
     description="Add the user in the request params to a group.",
 )
-def add_member(context: GroupContext, request):
+def add_member(context: GroupMembershipContext, request):
```

Comment:
`GroupMembershipContext` is backwards-compatible with `GroupContext` so no changes should be needed to `add_member()` or to any permissions predicate functions mapped to the `MEMBER_ADD` permission that `add_member()` uses.